core/src/main/scala/com/salesforce/op/dsl/RichMapFeature.scala

```diff
@@ -322,6 +328,8 @@ trait RichMapFeature {
         .setHashSpaceStrategy(hashSpaceStrategy)
         .setHashAlgorithm(hashAlgorithm)
         .setBinaryFreq(binaryFreq)
+        .setTokenizeForLengths(tokenizeForLengths)
```

Comment:
can we make this an enum rather than a boolean? then we have room to expend in the future